tests/integration/pure_cairo_vm/cairo1_integration/data/cairo_project.toml

```diff
@@ -0,0 +1,2 @@
+[crate_roots]
```

Comment:
Why do we need to use shared `data` if we have only one test file?

Reply:
Karol suggested to have it in a more "common" space

Reply:
M suggestion was to have it in the folder with the same name as the test but I'm also ok with having it in the `data` folder. The suggestion's [here](https://github.com/software-mansion/protostar/pull/1522#discussion_r1116822352).

Originally it was stored in the variable, so I think it's better to have it in a file. My suggestion wasn't precisely about having it in a more "common" space, that wasn't what I had in mind exactly.